contributing/writing-css.md

```diff
@@ -0,0 +1,204 @@
+# Writing CSS For AMP Runtime?
+
+- [Who should read this doc?](#who-should-read-this-doc)
+- [What is Specificity?](#what-is-specificity)
+  - [Example CSS selectors](#example-css-selectors)
+  - [How to calculate Specificity](#how-to-calculate-specificity)
+- [Writing CSS in AMP Runtime or Extensions](#writing-css-in-amp-runtime-or-extensions)
+  - [Creating a new Selector:](#creating-a-new-selector)
+  - [Modifying an existing Selector:](#modifying-an-existing-selector)
+  - [FOUC](#fouc)
+  - [Rule of thumb](#rule-of-thumb)
+- [Case Studies](#case-studies)
+  - [AMP Selector CSS has high CSS specificity](#amp-selector-css-has-high-css-specificity)
+  - [AMP Accordion CSS Changes broke specificity](#amp-accordion-css-changes-broke-specificity)
+- [Reading material](#reading-material)
+
+## Who should read this doc?
+
+Are you a contributor to ampproject/amphtml, so you make changes to the runtime or add or
```

Comment:
Either:

- Are you a contributor to ampproject/amphtml? Do you make changes to the AMP Runtime or add/modify AMP extensions? If you answer yes, then you should definitely read this document.
- Do you contribute changes to the AMP Runtime or add/modify AMP extensions? If you answer yes, then you should definitely read this document.